reference/5.0/CimCmdlets/CimCmdlets.md

```diff
@@ -1,51 +1,65 @@
 ---
-ms.date:  06/09/2017
+ms.date:  02/20/2019
 schema:  2.0.0
 Module Name: CimCmdlets
 keywords:  powershell,cmdlet
 Help Version:  5.0
 Module Guid: fb6cc51d-c096-4b38-b78d-0fed6277096a
-Download Help Link: {{Please enter FwLink manually}}
+Download Help Link:
 Locale: en-US
 ---
-
 # CimCmdlets Module
+
 ## Description
-{{Manually Enter Description Here}}
+
+Contains cmdlets that interact with CIM Servers like the WMI service in Windows.
```

Comment:
CIM and WMI should both be spelled out and probably links to more info in first use.

Reply:
Expanded the TLAs but didn't link. Assume that they know why they are here.